examples/petstore/models/Pet.go

```diff
@@ -20,6 +21,7 @@ type Pets struct {
 	Age        int        `json:"age" example:"2" description:"Age of the pet, in years"`
 	IsAdopted  bool       `json:"is_adopted" description:"Is the pet adopted"`
 	References References `json:"references"`
+	BornDate   time.Time  `json:"born_date"`
```

Comment:
I think people would refer as birth date


```suggestion
	BirthDate   time.Time  `json:"birth_date"`
```

Reply:
I feel dumb